src/core/xref.js

```diff
@@ -72,6 +84,71 @@ export async function run(conf, elems) {
   addDataCiteToTerms(results, xrefMap, conf);
 }
 
+/**
+ * converts conf.xref to object with url and spec properties
+ */
+function normalizeConfig(xref) {
+  const xrefDefaults = {
+    url: API_URL,
+    specs: null,
+  };
+
+  const normalizedConfig = {};
+  const type = Array.isArray(xref) ? "array" : typeof xref;
+  switch (type) {
+    case "boolean":
+      // ... true
+      Object.assign(normalizedConfig, xrefDefaults);
+      break;
+    case "string":
+      if (profiles[xref]) {
+        // We have the profile
+        Object.assign(normalizedConfig, xrefDefaults, {
+          specs: profiles[xref],
+        });
+      } else {
+        pub(
+          "error",
+          `Invalid cross-references profile "${xref}". Please use one of the supported specification profiles.`
+        );
+        Object.assign(normalizedConfig, xrefDefaults);
+      }
+      break;
+    case "array":
+      Object.assign(normalizedConfig, xrefDefaults, {
+        specs: [].concat(xref),
+      });
+      break;
+    case "object":
+      if (xref.profile && !profiles[xref.profile]) {
+        pub(
+          "error",
+          `Invalid cross-references profile "${
+            xref.profile
+          }". Please use one of the supported specification profiles.`
```

Comment:
As above... you might want to make this a function to avoid code duplication.

Reply:
Yes, I was thinking of that. Will do.